rtsp_to_hls/hls_proxy_api/lib/connection/rtsp_keep_alive.ex

```diff
@@ -0,0 +1,33 @@
+defmodule HlsProxyApi.Connection.RtspKeepAlive do
+  @moduledoc """
+  This module is responsible for maintaining the RTSP connection.
+  """
+  use GenServer
+
+  require Logger
+
+  alias Membrane.RTSP
+
+  @spec start(pid()) :: GenServer.on_start()
+  def start(session) do
+    GenServer.start(__MODULE__, session)
+  end
+
+  @impl true
+  def init(session) do
+    Process.send_after(self(), :ping, 15_000)
```

Comment:
We have better parametrize that `15_000` since it appears more than one (how about adding a `@delay` tag?)

Reply:
Done